Automattic/jetpack
# DNA: Move some root-level files to the jetpack-standards repo

#### Changes proposed in this Pull Request:
* Move some dev environment files to a new composer package for others to also use.

Note: Does not run in production so it doesn't have to wait for Jetpack's PHP version bump.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Part of the DNA.

#### Testing instructions:
* Run `composer install` and verify the files deleted in this PR are added. They should not be in git.

#### Proposed changelog entry for your changes:
* Developer Experience: Use the new automattic/jetpack-standards package for setting up devex.
